### ISSUE TITLE
Set-DbaStartupParameter: Issue configuring trace flags on new instance [Cannot convert value "-T1118" to type "System.Int32"]

### ISSUE THREAD

**Issue by Geo-Ron** [sqlcollaborative/dbatools#7939]
### Verified issue does not already exist?

Yes

### What error did you receive?

```powershell
 Set-DbaStartupParameter -Credential $Credential -SqlInstance $Instance -TraceFlag 7412 -Confirm:$false
```
WARNING: [17:02:33][Set-DbaStartupParameter] Unable to gather current startup parameters | Error connecting to [<INSTANCE>]: Cannot convert value "-T1118" to type "System.Int32". Error: "Input string was not in a correct format."

 

### Steps to Reproduce

```powershell
 Set-DbaStartupParameter -Credential $Credential -SqlInstance $Instance -TraceFlag 7412 -Confirm:$false
 Set-DbaStartupParameter -Credential $Credential -SqlInstance $Instance -TraceFlagOverride
```

WARNING: [17:21:03][Set-DbaStartupParameter] Unable to gather current startup parameters | Error connecting to [<INSTANCE>]: Cannot convert value "<First TraceFlag Configured>" to type "System.Int32". Error: "Input string was not in a correct format."

### Are you running the latest release?

No, this is my bad. I was using 1.1.24, but the latest resolves this.

### Other details or mentions

_No response_

### What PowerShell host was used when producing this error

Windows PowerShell (powershell.exe)

### PowerShell Host Version


Name                           Value                                                                                                                                                                                                                                          
----                           -----                                                                                                                                                                                                                                          
PSVersion                      5.1.17763.2183                                                                                                                                                                                                                                 
PSEdition                      Desktop                                                                                                                                                                                                                                        
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                                                                        
BuildVersion                   10.0.17763.2183                                                                                                                                                                                                                                
CLRVersion                     4.0.30319.42000                                                                                                                                                                                                                                
WSManStackVersion              3.0                                                                                                                                                                                                                                            
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                                                            
SerializationVersion           1.1.0.1                                                                                                                                                                                                                                        




### SQL Server Edition and Build number

Microsoft SQL Server 2019 (RTM-CU13) (KB5005679) - 15.0.4178.1 (X64)   Sep 23 2021 16:47:49   Copyright (C) 2019 Microsoft Corporation  Standard Edition (64-bit) on Windows Server 2019 Standard 10.0 <X64> (Build 17763: ) (Hypervisor) 

### .NET Framework Version
```powershell
PS Microsoft.PowerShell.Core\FileSystem::\\t...\DBA-Setup> (Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full").Release
461814
```

**Comment by Geo-Ron** [comment]
Issue has been resolved in 1.1.34.
The issue was in Get-DbaStartupParameter

I am sorry for creating this issue.

**Comment by potatoqualitee** [comment]
No problem, glad you found the fix!
